<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HexForm</class>
 <widget class="QWidget" name="HexForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>48</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lb_columnGrouping">
       <property name="text">
        <string>Column Grouping</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>sb_columnGrouping</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sb_columnGrouping">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="value">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_fontSize">
       <property name="text">
        <string>Font Size</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>hs_fontSize</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="hs_fontSize">
       <property name="minimum">
        <number>4</number>
       </property>
       <property name="maximum">
        <number>24</number>
       </property>
       <property name="pageStep">
        <number>2</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_showHeaders">
       <property name="text">
        <string>Show Headers</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
